mqtt/forwarder.go

```diff
@@ -20,7 +21,7 @@ const (
 type Forwarder interface {
 	// Forward subscribes to the Subscriber and
 	// publishes messages using provided Publisher.
-	Forward(id string, sub messaging.Subscriber, pub messaging.Publisher) error
+	Forward(ctx context.Context, id string, sub messaging.Subscriber, pub messaging.Publisher) error
```

Comment:
We can replace the variable `ctx` with `_`, since it not used inside the function